cvat/apps/documentation/user_guide.md

```diff
@@ -1,232 +1,446 @@
+    - ``git@gitlab-icv.inn.intel.com:project/repos[.git]``
+
+    The task will be highlighted in red after creation if annotation isn't synchronized with the repository.
+
+    **Use LFS**. If the annotation file is large, you can create a repository with
+    [LFS](https://git-lfs.github.com/) support.
+
+    **Source**. To create huge tasks please use ``shared`` server directory (choose ``Share`` option in the dialog).
+
+    **Z-Order**. Defines the order on drawn polygons. Check the box for enable layered displaying.
+
+    **Overlap Size**. Use this option to make overlapped segments.
+    The option makes tracks continuous from one segment into another.
+    Use it for interpolation mode. There are several options for using the parameter:
+    - For an interpolation task (video sequence).
+    If you annotate a bounding box on two adjacent segments they will be merged into one bounding box.
+    If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file,
+    you will have several tracks, one for each segment, which corresponds to the object.
+    - For an annotation task (independent images).
+    If an object exists on overlapped segments, the overlap is greater than zero
+    and the annotation is good enough on adjacent segments, it will be automatically merged into one object.
+    If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file,
+    you will have several bounding boxes for the same object.
+    Thus, you annotate an object on the first segment.
+    You annotate the same object on second segment, and if you do it right, you
+    will have one track inside the annotations.
+    If annotations on different segments (on overlapped frames)
+    are very different, you will have two shapes for the same object.
+    This functionality works only for bounding boxes and polygons
+    Polyline, points don't support automatic merge on overlapped segments
+    even the overlap parameter isn't zero and match between corresponding shapes on adjacent segments is perfect.
+
+    **Segment size**. Use this option to divide a huge dataset into a few smaller segments.
+    For example, one job cannot be annotated by several labelers (it isn't supported).
+    Thus using "segment size" you can create several jobs for the same annotation task.
+    It will help you to parallel data annotation process.
+
+    **Image Quality**. Use this option to specify quality of uploaded images.
+    The option helps to load high resolution datasets faster.
+    Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
+
+    **Select files**. Push this button to select files you want to annotate.
+
+    Push ``Submit`` button and it will be added into the list of annotation tasks.
+    Then, the created task will be displayed on dashboard:
+
+    ![](static/documentation/images/image006.jpg)
```

Comment:
The image contains very strange URL. Need to blur it (only ?id=...) should be visible.